integration-tests/pom.xml

```diff
@@ -190,6 +195,7 @@
                                 -Dfile.encoding=UTF-8
                                 -Dtestrunfactory=org.testng.DruidTestRunnerFactory
                                 -Ddruid.test.config.dockerIp=${env.DOCKER_IP}
+                                -Ddruid.test.config.hadoopDir=${env.HADOOP_DIR}
```

Comment:
I've modified issue https://github.com/druid-io/druid/issues/2531, which was for a batch indexing test, to be for being able to run such a test with docker.  

Any other comments on this PR?